.eslintrc

```diff
@@ -0,0 +1,164 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true // Allows support of JSX, but use of React plugin is required to support React semantics
+    }
+  },
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
+    "node",
+    "promise",
+    "react",
+    "@typescript-eslint"
+  ],
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  },
+  "env": {
+    "amd": true,
+    "browser": true,
+    "jquery": true,
+    "node": true,
+    "es6": true, // This enables ES6 global variables AND ES6 syntax
+    "worker": true
+  },
+  "rules": {
+    // The below are some, but not all, of the rules from eslint:recommended https://github.com/eslint/eslint/blob/master/conf/eslint-recommended.js (all errors set to warning)
```

Comment:
Why not report errors? will it break the build?

Reply:
Error here means the severity that is reported to the user